formats/shared/definitions/_specialist_document.jsonnet

```diff
@@ -1955,6 +1958,286 @@
+            "JP",
+            "JO",
+            "KZ",
+            "KE",
+            "KI",
+            "XK",
+            "KW",
+            "KG",
+            "LA",
+            "LV",
+            "LB",
+            "LS",
+            "LR",
+            "LY",
+            "LI",
+            "LT",
+            "LU",
+            "MK",
+            "MG",
+            "MW",
+            "MY",
+            "MV",
+            "ML",
+            "MT",
+            "MH",
+            "MR",
+            "MU",
+            "MX",
+            "FM",
+            "MD",
+            "MC",
+            "MN",
+            "ME",
+            "MA",
+            "MZ",
+            "NA",
+            "NR",
+            "NP",
+            "NL",
+            "NZ",
+            "NI",
+            "NE",
+            "NG",
+            "KP",
+            "NO",
+            "OM",
+            "PK",
+            "PW",
+            "PA",
+            "PG",
+            "PY",
+            "PE",
+            "PH",
+            "PL",
+            "PT",
+            "QA",
+            "RO",
+            "RU",
+            "RW",
+            "WS",
+            "SM",
+            "ST",
+            "SA",
+            "SN",
+            "RS",
+            "SC",
+            "SL",
+            "SG",
+            "SK",
+            "SI",
+            "SB",
+            "SO",
+            "ZA",
+            "KR",
+            "SS",
+            "ES",
+            "LK",
+            "KN",
+            "LC",
+            "VC",
+            "SD",
+            "SR",
+            "SZ",
+            "SE",
+            "CH",
+            "SY",
+            "TJ",
+            "TZ",
+            "TH",
+            "TG",
+            "TO",
+            "TT",
+            "TN",
+            "TR",
+            "TM",
+            "TV",
+            "UG",
+            "UA",
+            "AE",
+            "GB",
+            "US",
+            "UY",
+            "UZ",
+            "VU",
+            "VA",
+            "VE",
+            "VN",
+            "YE",
+            "ZM",
+            "ZW",
+          ],
+        },
+      },
+      fcdo_authors: {
+        type: "array",
+        items: {
+          type: "string",
+        },
+      },
+      first_published_at: {
+        type: "string",
+        pattern: "^[1-9][0-9]{3}[-/](0[1-9]|1[0-2])[-/](0[1-9]|[12][0-9]|3[0-1])$",
+      },
+      fcdo_theme: {
+        type: "array",
+        items: {
+          type: "string",
+          enum: [
+            "agriculture",
+            "climate_and_environment",
+            "economic_growth",
+            "education",
+            "food_and_nutrition",
+            "governance_and_conflict",
+            "health",
+            "humanitarian_disasters_and_emergencies",
+            "infrastructure",
+            "research_communication_and_uptake",
+            "social_change",
+            "water_and_sanitation",
+          ],
+        },
+      },
+      fcdo_document_type: {
```

Comment:
I guess this one could be problematic because of it's name... Maybe `research_document_type`?